wagtail/snippets/tests/test_viewset.py

```diff
@@ -769,6 +769,22 @@ def test_boolean_column(self):
         )
 
 
+class TestRelatedFieldColumns(BaseSnippetViewSetTests):
+    model = SnippetChooserModel
+
+    def test_snippetchoosermodel_related_fields(self):
```

Comment:
Could you add another test case where the foreign key is null? e.g.

```py
    def test_snippetchoosermodel_related_fields(self):
        advert = Advert(url="https://example.com/free_examples", text="Free Examples")
        advert.save()
        scm = SnippetChooserModel(advert=advert)
        scm.save()

        response = self.client.get(self.get_url("list"))

        self.assertEqual(response.status_code, 200)
        # assert the header label here
        # (and maybe assert that the cell is empty)
```

Right now it seems to crash as the `multigetattr` function tries to access `None["text"]`. You may have to add a `try`/`except` in `Column.get_value()` to return `None` when this occurs.

Reply:
Done